docs/man/man1/precice-tools.1

```diff
@@ -34,7 +34,7 @@ Prints the preCICE version string ( \fIprecice::getVersionInformation()\fR ) to
 
 .TP
 .B check \fIfile\fR [\fIparticipant\fR [\fIsize\fR]]
-Checks given preCICE configuration \fIfile\fR and prints any errors. Pass the name of a \fIparticipant\fR to fully check its configuration. The optional \fIsize\fR indicates how many ranks run in parallel, which checks custom \fI<primary-rank: ... />\fR tags.
+Checks given preCICE configuration \fIfile\fR and prints any errors. Pass the name of a \fIparticipant\fR to fully check its configuration. The optional \fIsize\fR indicates how many ranks run in parallel, which checks custom \fI<on-primary-rank: ... />\fR tags.
```

Comment:
`intra-comm`, not `on-primary-rank`

Reply:
Fixed in https://github.com/precice/precice/pull/1283